apps/base-docs/docs/pages/wallet-app/mini-apps.mdx

```diff
@@ -0,0 +1,102 @@
+
+As described below we don't think it's the best practice to prompt the user to log in before that authentication will allow them to do something else in your mini app, but for cases where do you want a secure, persisted session, using a wallet connection is a great option.
+
+### Context Data
+
+All mini app host apps(including Coinbase Wallet) return [context data](https://miniapps.farcaster.xyz/docs/sdk/context), which tells developers about the app/mini app host the user is accessing their mini app in, as well as **which user** is interacting with their mini app.
+
+While this data can technically be spoofed, not only can you be sure that Coinbase Wallet will always give you the accurate data as a developer but also making use of this context data is super helpful because it doesn't require any other action from the user.
+
+A common flow that other mini app developers follow is using the context data to either track analytics metrics or create an authentication session via a [JWT](https://jwt.io/introduction). This allows you to still track analytics/offer authenticated services to your users without the extra friction of signing a message or deeplinking to another app.
+
+### Sign In with Farcaster
+
+Currently, using Sign In with Farcaster inside of Coinbase Wallet is **not recommended as the primary method of authentication**. This is because for Farcaster accounts that were created in Warpcast(which many current accounts were), using Sign In with Farcaster will require deeplinking the user to Warpcast and then back to Coinbase Wallet. While this flow still works inside of Coinbase Wallet, we recommend either wallet auth or creating an auth strategy around the context data(eg. a custom JWT with context data) for a more optimal UX.
+
+Note: The Sign In with Farcaster flow will no longer require a deeplink to Warpcast when the [auth addresses FIP](https://github.com/farcasterxyz/protocol/discussions/225) lands, which will allow a set of delegated wallets to take actions on behalf of a Farcaster account's custody wallet.
+
+## Deeplinks and SDK Actions
+
+The official mini apps SDK offers a [set of actions](https://miniapps.farcaster.xyz/docs/specification#actions)(which MiniKit offers as well) so that users of your mini app can be led to do things back in clients like Coinbase Wallet(eg. compose a cast, view a profile, etc).
+
+Although a few other mini app developers have used the workaround of inserting Warpcast-specific deeplinks in the `openUrl` function of the SDK for certain actions, **it is highly recommended that you make use of all official SDK functions instead of using Warpcast-specific deeplinks in your mini app**. This is because certain Warpcast deeplinks might lead to dead ends/unpleasent experiences, and using SDK functions instead will ensure your users have the best viewing experience possible in Coinbase Wallet.
```

Comment:
can we give any more detail to "certain Warpcast deeplinks"? which kind? what kind of dead end / unpleasant experience?